x/examples/website-wrapper-app/.scripts/lib/constants.mjs

```diff
@@ -0,0 +1,25 @@
+import path from "node:path";
+
+export const OUTPUT_DIR = path.join(process.cwd(), "output");
+
+export const NAVIGATION_PROXY_DIR = path.join(process.cwd(), "navigation_proxy");
```

Comment:
Please don't mix the navigator with the app maker. That is something that goes on the site, not the app.

I suggest just moving it out of here, into the start script.